src/components/MoneyTemporaryForRefactorRequestConfirmationList.js

```diff
@@ -756,7 +768,49 @@ function MoneyTemporaryForRefactorRequestConfirmationList({
                     interactive={!isReadOnly}
                 />
             ),
-            shouldShow: isDistanceRequest,
+            shouldShow: isDistanceRequest && !canUseP2PDistanceRequests,
+            isSupplementary: true,
```

Comment:
I don't think we want to change this to a supplementary field, for example if you can't use p2p distance requests it should still show above the fold.